### ISSUE TITLE
Tracking: cube.js support

### ISSUE THREAD

**Issue by xiangjinwu** [risingwavelabs/risingwave#9946]
https://github.com/cube-js/cube
https://cube.dev/docs/getting-started/core/create-a-project

Current issues:
* `SET TIME ZONE 'UTC'` https://github.com/risingwavelabs/risingwave/pull/8572#discussion_r1138158571
* `GROUP BY 1` #4681
* (will add more progressively)

**Comment by Boburmirzo** [comment]
Yeah, for RisingWave we need a separate driver implemented as Materiliaze did:

https://github.com/cube-js/cube/pull/4320
